tests/mysql/query.rs

```diff
@@ -540,27 +540,27 @@ fn select_36() {
 
 #[test]
 fn select_37() {
-    let (statement, values) = sea_query::Query::select()
+    let (statement, values) = Query::select()
         .column(Glyph::Id)
         .from(Glyph::Table)
         .cond_where(Cond::any().add(Cond::all()).add(Cond::any()))
-        .build(sea_query::MysqlQueryBuilder);
+        .build(MysqlQueryBuilder);
 
-    assert_eq!(statement, r#"SELECT `id` FROM `glyph`"#);
+    assert_eq!(statement, r#"SELECT `id` FROM `glyph` WHERE FALSE"#);
```

Comment:
I think there's some unfortunate interaction between 2 features here:
 - Collapsing an empty any/all into a boolean
 - Not adding an any/all that's empty.

This should be `any(TRUE, FALSE)`, so `TRUE`

Reply:
Fair

Reply:
Update: I am try to understand why it output `FALSE` here:

I imagine the expected output to be:

```SQL
SELECT `id` FROM `glyph` WHERE TRUE OR FALSE
```

Reply:
Ah found it